addons/website/views/snippets/s_popup.xml

```diff
@@ -85,7 +85,8 @@
                 <we-button data-move-block="moveToFooter">All pages</we-button>
             </we-select>
         </t>
-        <div data-js="SnippetPopup" data-selector=".s_popup" data-exclude="#website_cookies_bar" data-target=".modal">
+        <div data-js="SnippetPopup" data-selector=".s_popup" data-exclude="#website_cookies_bar" data-target=".modal"
+            data-exclude-drop-in=".oe_structure .oe_structure">
```

Comment:
It seems like a good idea to add a data-exclude-drop-in attribute, the drop-in selectors will be more clearly defined :+1: